modules/end-user-guide/nav.adoc

```diff
@@ -3,6 +3,7 @@
 * xref:navigating-che.adoc[]
 ** xref:navigating-che-using-the-dashboard.adoc[]
 ** xref:importing-certificates-to-browsers.adoc[]
+** xref:using-private-container-registries.adoc[]
```

Comment:
I'm not sure Navigating Che is the best place for it.
We may want to have dedicated sections for Using Workspace and Configuring workspace, and then it may be part of Configuring workspaces.

Reply:
I agree that it is not the best place and would deserve a new section. Maybe we could create an issue for that.

Reply:
Then in the section under 

```
* xref:customizing-developer-environments.adoc[]
```